scripts/deploy/07_steth_logic.ts

```diff
@@ -11,6 +11,7 @@ import {
   STETH_ETH_CRV_POOL,
   WSTETH_ADDRESS,
 } from "../../constants/constants";
+import { PREMIUM_DISCOUNT, AUCTION_DURATION } from '../utils/constants';
 
```

Comment:
not adding to constructor, but to init. should be in 08_steth_call_theta_vault.ts

Reply:
should be right below here https://github.com/ribbon-finance/ribbon-v2/blob/4fcd7ba72c86c23d9e3081a005c2aa7a1226b787/scripts/deploy/08_steth_call_theta_vault.ts#L55

Reply:
Fixed